.github/workflows/release.yml

```
@@ -29,22 +25,21 @@ on:
         required: false
         description: If true, the release will not be created.
         default: false
-        type: boolean
+        type: boolean          
```

Remove extra whitespace

```suggestion
        type: boolean
```